### PR TITLE
checkout page- headless - IHS-14

### PR DESCRIPTION
**Pull request by prajaktaIgn**
chedckout page changes integrated

### CODE REVIEW

File: components/checkout/OrderReview/OrderReview.tsx [hunk 1]
```diff
@@ -167,6 +167,7 @@ const OrderReview = (props: OrderReviewProps) => {
       <AccordionDetails data-testid="accordion-details" sx={{ ...style.accordionDetails }}>
         <Divider sx={{ marginX: '20px' }} />
         <StyledOrderReview>
+          {/*
           <OrderInfoHeader
             headerName={t('personal-details')}
```
Comment (mayurk1320) [inline components/checkout/OrderReview/OrderReview.tsx]: Remove commented code

File: components/checkout/OrderReview/OrderReview.tsx [hunk 2]
```diff
@@ -177,46 +178,52 @@ const OrderReview = (props: OrderReviewProps) => {
               <Typography variant="body1">{shippingPhoneHome}</Typography>
             </Box>
           </OrderInfoHeader>
+      */}
+          {billingAddress?.address1 !== shippingAddress?.address1 &&
+            billingAddress?.address2 !== shippingAddress?.address2 &&
+            billingAddress?.cityOrTown !== shippingAddress?.cityOrTown &&
+            billingAddress?.stateOrProvince !== shippingAddress?.stateOrProvince &&
```
Comment (mayurk1320) [inline components/checkout/OrderReview/OrderReview.tsx:173]: Why do we have so many checks? See, if we can refactor it.

File: components/checkout/ReviewStep/ReviewStep.tsx [hunk 3]
```diff
@@ -114,6 +115,9 @@ const ReviewStep = (props: ReviewStepProps) => {
   const [isAgreeWithTermsAndConditions, setAgreeWithTermsAndConditions] =
     useState<boolean>(isAuthenticated)
 
+  console.log('personalDetails:', personalDetails)
+  console.log('checkout:', checkout)
```
Comment (mayurk1320) [inline components/checkout/ReviewStep/ReviewStep.tsx]: Remove console

File: components/checkout/ReviewStep/ReviewStep.tsx [hunk 4]
```diff
@@ -178,10 +182,10 @@ const ReviewStep = (props: ReviewStepProps) => {
   const handleComplete = () => handleSubmit(onValid, onInvalidForm)()
 
   const orderPriceProps: OrderPriceProps = {
-    subTotalLabel: t('subtotal'),
-    shippingTotalLabel: t('shipping'),
-    taxLabel: t('estimated-tax'),
-    totalLabel: t('total'),
+    subTotalLabel: t('Sub-Total :'),
+    shippingTotalLabel: t('Delivery Fee :'),
```
Comment (mayurk1320) [inline components/checkout/ReviewStep/ReviewStep.tsx]: add static text in translation file

File: components/checkout/ReviewStep/ReviewStep.tsx [hunk 5]
```diff
@@ -399,6 +425,12 @@ const ReviewStep = (props: ReviewStepProps) => {
         )}
       </Box>
 
+      <Stack sx={{ mt: '20px', mb: '20px' }}>
+        {t(
+          'Please review your order below.Your order is not final until you click the Place Order button.'
```
Comment (mayurk1320) [inline components/checkout/ReviewStep/ReviewStep.tsx]: add static text in translation file

**Review by mayurk1320 — CHANGES_REQUESTED** [review]

File: components/checkout/OrderReview/OrderReview.tsx (hunk 2 above)
Comment (prajaktaIgn) [reply components/checkout/OrderReview/OrderReview.tsx:173]: billingAddress and shipping address is object we cannot check one field over here we need to verify all the fields so if shipping and billing address is same then only billing address will be display

**Pull request merged**